src/store/config/index.ts

```diff
@@ -54,7 +54,9 @@ export const defaultState = (): ConfigState => {
         zAdjustDistances: [0.005, 0.01, 0.025, 0.050],
         enableVersionNotifications: true,
         confirmOnEstop: false,
-        confirmOnPowerDeviceChange: false
+        confirmOnPowerDeviceChange: false,
+        dateformat: 'MMM. DD',
+        timeformat: 'h:mm a'
```

Comment:
@pedrolamas  What about defaulting to ISO 8601 for datetime formats?

    2022-01-26T20:11:23Z

Reply:
@Biorn1950 good point (about how existing works/looks right now).  I was just thinking of the hardcoded values defaulting above, which now that I think about it is user-facing, not backend-used.

So yeah, ignore my comment.